controllers/git_test.go

```diff
@@ -128,21 +132,26 @@ func TestPushRejected(t *testing.T) {
 	}
 
 	// this is currently defined in update_test.go, but handy right here ..
-	if err = initGitRepo(gitServer, "testdata/appconfig", "main", "/appconfig.git"); err != nil {
+	if err = initGitRepo(gitServer, "testdata/appconfig", "test", "/appconfig.git"); err != nil {
```

Comment:
Would be nice to capture the reason for using a different branch in commit description.

Reply:
adding a comment instead